src/lib.rs

```diff
@@ -141,6 +141,16 @@ pub mod corebluetooth;
 #[cfg(target_os = "windows")]
 pub mod winrtble;
 
+#[allow(unused_imports)]
+#[cfg(target_os = "linux")]
+pub use crate::bluez::{manager::Manager, adapter::{ConnectedAdapter, Adapter}};
+#[allow(unused_imports)]
+#[cfg(target_os = "macos")]
+pub use crate::corebluetooth::{manager::Manager, adapter::{Adapter, Adapter as ConnectedAdapter}};
+#[allow(unused_imports)]
+#[cfg(target_os = "windows")]
+pub use crate::winrtble::{manager::Manager, adapter::{Adapter, Adapter as ConnectedAdapter}};
+
```

Comment:
Rather than putting this at the top level, how about putting the re-exports under a new module, called 'platform' or something like that?

This module could also have some tests to ensure that the various implementations implement the interfaces we expect them to (both the interfaces under api, and things like `Debug` or `Clone` or whatever else we expect).